src/main/scala/cromwell/backend/provider/local/LocalBackend.scala

```diff
@@ -105,7 +106,7 @@ class LocalBackend(task: TaskDescriptor) extends BackendActor with StrictLogging
     * Executes task in given context.
     */
   override def execute(): Unit = {
-    notifyToSubscribers(executeTask)
+    notifyToSubscribers(executeTask())
```

Comment:
I did this to clear up the yellow from the IntelliJ default settings which warn on empty paren methods invoked without parens.